packages/create-vite/template-vue-ts/src/App.vue

```diff
@@ -1,6 +1,4 @@
 <script setup lang="ts">
-// This starter template is using Vue 3 <script setup> SFCs
-// Check out https://vuejs.org/api/sfc-script-setup.html#script-setup
```

Comment:
I think we can remove this.
This is written in README and `<script setup>` is released more than a year ago. (2021/08)